### PR TITLE
chore: upgrade node to 18

### PR DESCRIPTION
**Pull request by gergelypap**
# Brief description

I've upgraded the project to require node 18, and allow usage with npm 9. Also updated CI to use 18 only.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -9,7 +9,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [16.x, 18.x]
```
Comment (csutorasr) [inline .github/workflows/ci.yaml:12]: Can't we continue support node 16?

File: README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ The text reporter writes `probable root cause` text to the output, if that packa
 
 ## NPM version support
 
-This package supports NPM 7 and 8 from `2.0.0`. If you want to use it with NPM 6, then install the latest [`1.x`](https://github.com/team-supercharge/audit-ci-wrapper/tree/v1.x) version.
+This package requires at least NPM 7 since `2.0.0`. If you want to use it with NPM 6, then install the latest [`1.x`](https://github.com/team-supercharge/audit-ci-wrapper/tree/v1.x) version.
```
Comment (csutorasr) [inline README.md:17]: The 2.0.0 version is not true. Update it to `>3.0.0`.

**Review by csutorasr — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (gergelypap) [reply README.md:17]: We are requiring npm versions in 2.0.0:

https://github.com/team-supercharge/audit-ci-wrapper/compare/v1.0.2...v2.0.0#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (gergelypap) [reply .github/workflows/ci.yaml]: Sure, I put it back.

File: README.md (hunk 2 above)
Comment (csutorasr) [reply README.md:17]: We require npm 7 and npm 8 there. From now on (in the next release = version bigger than 3.0.0) we require npm 7-8-9.

Comment (gergelypap) [reply README.md:17]: image

**Review by csutorasr — APPROVED** [review]

**Pull request merged**